pkgs/development/python-modules/terminaltables3/default.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  poetry-core,
+}:
+
+buildPythonPackage rec {
+  pname = "terminaltables3";
+  version = "4.0.0";
+  pyproject = true;
+
+  propagatedBuildInputs = [
+    poetry-core
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Tj7v4gmqiQBaCjTRUlc5QkVpcp7im15kqN1Rxevat38=";
+  };
+
+  meta = with lib; {
+    description = "Display simple tables in terminals";
+    homepage = "https://github.com/matthewdeanmartin/terminaltables3";
+    license = licenses.mit;
```

```suggestion
  meta = {
    description = "Display simple tables in terminals";
    homepage = "https://github.com/matthewdeanmartin/terminaltables3";
    license = lib.licenses.mit;
```
New packages need a maintainer

---

I'm kind of just adding this in parallel to the other package that's archived, to which this is the drop in replacement to. I was actually considering to just update the old definition but it sadly exports under a different namespace in python and I thought it's best to not break that compatibility for people.

This was really not an oversight but an active choice. If you really want me to maintain this, I can. But is that really necessary?